### PR TITLE
Store when a user last created a session

### PR DESCRIPTION
**Pull request by lozette**
## Changes

When a user logs in (creates a session), store a DateTime in the `latest_session` column on the user. This value will update each time a user creates a session.

## Checklist

- [x] Attach this pull request to the appropriate card in Trello.
- [x] Update the `CHANGELOG.md` if needed.

### CODE REVIEW

**Comment by lozette** [comment]
[Record the last sign in datetime for each session.](https://trello.com/c/JA7zkZKh/1890-record-the-last-sign-in-datetime-for-each-session)

**Review by mec — CHANGES_REQUESTED** [review]
Could we show the last session date and time on the users table at `/users/active` - sorry it wasn't on the story!

File: spec/helpers/user_helper_spec.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe UserHelper, type: :helper do
+  describe "#last_seen_datetime" do
+    context "with an active user" do
+      it "shows the formatted latest_session date" do
+        user = create(:user, latest_session: DateTime.new(2023, 0o1, 0o1, 10, 30, 0o0, 0))
```
Comment (mec) [inline spec/helpers/user_helper_spec.rb:7]: What are those dates (`0o1`?)

**Review by mec — CHANGES_REQUESTED** [review]
Not so much a change, as an explination of those date arguments - no idea what those are?

**Pull request merged**